config/jobs/kubernetes-sigs/cluster-api/templates/cluster-api-presubmits.yaml.tpl

```diff
@@ -453,4 +453,61 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api{{ if eq $.branch "main" | not -}}{{ TrimPrefix $.branch "release" }}{{- end }}
       testgrid-tab-name: capi-pr-e2e-conformance-ci-latest-{{ ReplaceAll $.branch "." "-" }}
+  - name: periodic-cluster-api-upgrade-{{ ReplaceAll (last $.config.Upgrades).From "." "-" }}-{{ ReplaceAll (last $.config.Upgrades).To "." "-" }}-{{ ReplaceAll $.branch "." "-" }}
+    cluster: eks-prow-build-cluster
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    extra_refs:
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
+      path_alias: k8s.io/kubernetes
+    decorate: true
+    decoration_config:
+      timeout: 180m
+    path_alias: sigs.k8s.io/cluster-api
+    always_run: false
+    branches:
+    # The script this job runs is not in all branches.
+    - ^{{ $.branch }}$
+    spec:
+      containers:
+      - image: {{ $.config.TestImage }}
+        args:
+        - runner.sh
+        - ./scripts/ci-e2e.sh
+        env:
+        # enable IPV6 in bootstrap image
+        - name: "DOCKER_IN_DOCKER_IPV6_ENABLED"
+          value: "true"
+        - name: GINKGO_SKIP
+          value: "\\[Conformance\\]"
+        # This value determines the minimum Kubernetes
+        # supported version for Cluster API management cluster
+        # and can be found by referring to [Supported Kubernetes Version](https://cluster-api.sigs.k8s.io/reference/versions.html#supported-kubernetes-versions)
+        # docs (look for minimum supported k8s version for management cluster, i.e N-3).
+        # Please also make sure to refer a version where a kindest/node image exists
+        # for (see https://github.com/kubernetes-sigs/kind/releases/)
```

Comment:
```suggestion
```
This comment doesn't make sense in this context

Reply:
Removed